cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/TimeScheduler.java

```diff
@@ -268,9 +264,22 @@ public void updateSchedule(Id.Program program, SchedulableProgramType programTyp
   @Override
   public void updateSchedule(Id.Program program, SchedulableProgramType programType, Schedule schedule,
                              Map<String, String> properties) throws NotFoundException, SchedulerException {
-    // TODO modify the update flow [CDAP-1618]
-    deleteSchedule(program, programType, schedule.getName());
-    schedule(program, programType, schedule, properties);
+    checkInitialized();
+    try {
+      Trigger trigger = getTrigger(program, programType, schedule.getName());
+      @SuppressWarnings("unchecked")
```

Comment:
this is not wrong. But this is done differently in schedule(). It would be nice follow the same style within the same file